jqassistant.properties

```diff
@@ -0,0 +1,21 @@
+category=External Analyzers
+description=Analyze issues reported by jQAssistant.
+homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
+archivedVersions=1.7.0
+publicVersions=1.7.1
+ 
+defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
+defaults.mavenArtifactId=sonar-jqassistant-plugin
+
+1.7.1.description=Reduced size of plugin JAR file
```

Comment:
This is the first version that will be available in the Marketplace. Far better version description would be something along the lines of "Initial entry into the Marketplace"